wagtail_wordpress_import/prefilters/fix_styles_filter.py

```diff
@@ -52,9 +31,25 @@ def fix_styles(value):
     later to decide if an element can have alignment in the richtext block
     e.g. "margin: 0pt 10px 0px 0pt; float: left;" maps to "leftfloat"
     e.g. "float: left; margin: 0em 1em 1em 0em;" maps to "leftfloat"
+
+    param: `options` NOT IMPLEMENTED
     """
-    soup = bs4(value, "html.parser")
-    search_styles = reverse_styles_dict(FILTER_MAPPING)
+
+    if options and options["CONFIG"].get("HTML_TAGS"):
+        # if options["CONFIG"].get("HTML_TAGS"):
+        function = import_string(options["CONFIG"]["HTML_TAGS"])
```

Comment:
An alternative to this could be:

```python
html_tags = import_string(options["CONFIG"]["HTML_TAGS"])
if callable(html_tags):
    CONF_HTML_TAGS = html_tags()
else:
    CONF_HTML_TAGS = html_tags
```

This removes the need for callables in constants